pkgs/top-level/python-aliases.nix

```diff
@@ -468,6 +468,7 @@ mapAliases ({
   pyruckus = throw "pyruckus has been removed, it was deprecrated in favor of aioruckus."; # added 2023-09-07
   py_scrypt = py-scrypt; # added 2024-01-07
   pysha3 = throw "pysha3 has been removed, use safe-pysha3 instead"; # added 2023-05-20
+  pysimplegui = throw "update to v5 broke the package, it now needs rsa and is trying to access an X11 socket?";
```

Comment:
```suggestion
  pysimplegui = throw "broke since v5 upgrade, code is obfuscated and needs a license to be used";
```

Reply:
Also please start with the package name so that people know about which package the throw is

Reply:
done